src/query/sql/src/executor/physical_plan_display.rs

```diff
@@ -82,9 +83,12 @@ impl<'a> Display for PhysicalPlanIndentFormatDisplay<'a> {
             PhysicalPlan::ProjectSet(unnest) => write!(f, "{}", unnest)?,
             PhysicalPlan::RuntimeFilterSource(plan) => write!(f, "{}", plan)?,
             PhysicalPlan::RangeJoin(plan) => write!(f, "{}", plan)?,
-            PhysicalPlan::DistributedCopyIntoTable(copy_into_table) => {
+            PhysicalPlan::DistributedCopyIntoTableFromStage(copy_into_table) => {
                 write!(f, "{}", copy_into_table)?
```

Comment:
copy_into_table_from_stage?

Reply:
yes, we have two cases, copy into table from stage like copy into t1 from @s1, secondly, copy into t from (select ....)

Reply:
> yes, we have two cases, copy into table from stage like copy into t1 from @s1, secondly, copy into t from (select ....)
we need to uniform these as one plan in the future. but they are not compatible now, from stage will be changed to from query by @youngsofun later.

Reply:
So if the display content should be "copy_into_table_from_stage".

Reply:
sure!!! thanks